### DISCUSSION TITLE
Compare Files in Linux (SCP connection)

### DISCUSSION THREAD

**Discussion by Jones1024** [WinMerge/winmerge#1046] [Ideas]
I run several Linux servers - without user interface. Is it possible to write a plugin for this?

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not support SCP.
It also doesn't have a mechanism like the VFS plugin.

Although I haven't tried it, WinMerge may be able to compare Linux files if you use software that makes the remote system appear as a network drive, such as WinFsp.

**Reply by Jones1024** [reply]
Thanks for the answer. Yes that with the network drives I currently do so. But it's not safe. Trojans, encrypt, etc.

**Comment by Jones1024** [comment]
Can any WinMerge developer say anything about this? Is SCP an interesting development for WinMerge?
